python/pyarrow/tests/test_csv.py

```diff
@@ -487,6 +487,27 @@ def test_trivial(self):
         table = self.read_bytes(rows)
         assert table.to_pydict() == {'': []}
 
+    def test_empty_lines(self):
+        rows = b"a,b\n\r1,2\r\n\r\n3,4\r\n"
+        table = self.read_bytes(rows)
+        assert table.to_pydict() == {
+            'a': [1, 3],
+            'b': [2, 4],
+            }
+        parse_options = ParseOptions(ignore_empty_lines=False)
+        table = self.read_bytes(rows, parse_options=parse_options)
+        assert table.to_pydict() == {
+            'a': [None, 1, None, 3],
```

Comment:
Why is there a None here as first element as well?

Reply:
Because `\n\r` is two newlines (hint: it's not `\r\n` ;-)).

Reply:
Ah, indeed, I was not seeing the difference between `\n\r` and `\r\n` ;)